### DISCUSSION TITLE
how to use custom classname with same name ("text")

### DISCUSSION THREAD

**Discussion by nicogulo** [tailwindlabs/tailwindcss#11108] [Help] [answered]
Sorry I came up with a strange discussion title. I'm currently working with a product designer, and he has created a design system.
On the design of the system, there is a config for color, where the name of the variable is the same as the tailwind className.
for example image below
![image](https://user-images.githubusercontent.com/63594315/234832328-9e196897-c705-41f4-ac3f-a010d9108424.png)

and I made a config like this. 
```
module.exports = {
  theme: {
    extend: {
      colors: {
         text: {
              main: "#3D4A5C",
        }
      }
    }
  },
  variants: {},
  plugins: [],
}
````
There's nothing strange about the configuration, but when using the styling color text it will feel strange because it uses the word "text" twice -> `text-text-main`

Is there a best practice way to configure it without changing the className?

**Comment by adamwathan** [comment, marked as answer]
You could configure your text colors under `textColor` directly:

https://play.tailwindcss.com/iKMD1rL6P6?file=config

**Reply by nicogulo** [reply]
Hi @adamwathan , thank you for this answer, actually this works for me. Thank you
